NitroxServer/ServerAutoFaqRegistrar.cs

```diff
@@ -0,0 +1,80 @@
+﻿using Autofac;
+using NitroxModel.Core;
+using System.Reflection;
+using NitroxServer.ConsoleCommands.Abstract;
+using NitroxServer.GameLogic;
+using NitroxServer.Communication;
+using NitroxServer.Serialization.World;
+using NitroxServer.GameLogic.Entities;
+using NitroxServer.Communication.Packets.Processors.Abstract;
+using NitroxServer.Communication.Packets;
+using NitroxServer.ConfigParser;
+using NitroxServer.ConsoleCommands.Processor;
+using NitroxServer.Communication.Packets.Processors;
+
+namespace NitroxServer
+{
+    public class ServerAutoFaqRegistrar : IAutoFacRegistrar
```

Comment:
Should be ServerAutoFa**c**Registrar

Reply:
I mimicked the name scheme in NitroxClient since over there its called ClientAutoFaqRegistrar

Reply:
Seems it was a typo because AutoFac is the name of the library and I have no idea what AutoFaq means.

Reply:
This was a typo in the initial implementation.

Reply:
Fixed now